index.html

```diff
@@ -15,27 +15,27 @@
   </head>
   <body>
     <div class="container">
-      <header>
+      <header class="navbar">
         <nav class="nav-bar">
           <a href="http://" class="alogo"> <h1 class="logo">My Logo</h1> </a>
           <ul class="nav-list">
-            <li class="nav-list-items">Portfolio</li>
-            <li class="nav-list-items">About</li>
-            <li class="nav-list-items">Contact</li>
+            <li class="nav-list-items"> <a href="#work" class="nav-list-items" id="portfoliolink"> Portfolio </a></li>
+            <li class="nav-list-items"><a href="#about" class="nav-list-items" id="aboutlink">About</a></li>
+            <li class="nav-list-items"><a href="#form" class="nav-list-items" id="contactlink">Contact</a></li>
           </ul>
           <img src="hamburger.svg" alt="hamburger-menu" class="nav-menu" />
```

Comment:
- [ ] I have noticed that after clicking on the humberger menu, and clicking on the cancel after, the humberger icon can no longer be displayed. Kindly ensure to display the icon at all the scenarios to have a professional and userfriendly portifolio.

Your design 😎 
![image](https://github.com/CrackedYoda/MyPortfolio/assets/99394374/6639251f-083f-4437-a3d8-40248a472cfb)